CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# Foodsoft 4.7.1
+(31 December 2020)
+
+* Fix minor navigation glitch ([78c4ebc](https://github.com/foodcoops/foodsoft/commit/78c4ebcb4b61891e13d3dbb9f07c34ad88c41f4a))
+* Fix legacy tasks without user ([beaca7d](https://github.com/foodcoops/foodsoft/commit/beaca7d384bf9c9a9c5275ac7d81bb012374372e))
+* Fix GroupOrder update timestamp ([3d5043f](https://github.com/foodcoops/foodsoft/commit/3d5043fd052f79b33f8cae2ea3a5cc81ee972ba5))
+* Move from Travis CI to GitHub Actions ([9b092ee](https://github.com/foodcoops/foodsoft/commit/9b092ee362df8a46854d8dc42125963f86dadf2f))
+* Fix database seeds ([#774](https://github.com/foodcoops/foodsoft/issues/774))
```

Comment:
this is also for developers and doesn't belong in the changelog

Reply:
> Need to bump the file VERSION too.

Right. This PR includes two commits. The second commit reverts the `VERSION` file. I also pushed the tag [v4.7.1-rc0](https://github.com/foodcoops/foodsoft/releases/tag/v4.7.1-rc0), and that tag refers to the commit that has `VERSION` set to 4.7.1.

> See also https://github.com/foodcoops/foodsoft/wiki/Creating-a-release - and since this is a small release, we may not need to do the full translations round, but it would be good if there are any changed strings.

I intentionally skipped the translations because I assumed that it is not required. Should I go back, run `script/crowdin-pull`, and commit?